ext/hal/st/stm32cube/stm32f4xx/README

```diff
@@ -43,3 +43,10 @@ Patch List:
     Impacted files:
      drivers/include/stm32f4xx_hal_conf.h
     ST Bug tracker ID: NA. Not a stm32cube issue
+
+   *Add correct shifting to I2SR field in PLLI2SCFGR register
+     The I2SR field should be shifted by RCC_PLLI2SCFGR_PLLI2SR_Pos when the PLLI2SCFGR register
+     is read or written.
+    Impacted files:
+     drivers/include/stm32f4xx_ll_rcc.h
+    ST Bug tracker ID: TO BE ASSIGNED
```

Comment:
50086

Reply:
>> Provided internal ST tracker.

ok, good.

>> Do you know if this bug is also present for other series

Well, no, I don't know.
I can check it quickly tomorrow. Let's say that the PLLI2S is not present in all the series if I got it properly. Maybe only stm32f4xx and stm32f10x. But I need to check carefully.